### PR TITLE
Add Feature request template

### LINKED ISSUE #3400: Add GitHub issue template for feature requests
We only have one issue template for bug reports right now. It would be nice to have an issue template for new features.

### PR DESCRIPTION
**Pull request by xlanor**
Added a feature request template which I think covers the bases.

This should not affect the changelog, as it does not affect Thanos code.

Closes #3400 

* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes
Added FEATURE_REQUEST.md under .github

### CODE REVIEW

File: .github/FEATURE_REQUEST.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+name: "Feature Request"
+about: Suggest an idea for improving Thanos
+title: ''
+labels: Enhancement
```
Comment (bwplotka) [inline .github/FEATURE_REQUEST.md:5]:
```suggestion
labels: "feature request / improvement"
```

**Review by bwplotka — APPROVED** [review]
Looks good! :muscle: 

Thanks, minor suggestion only.

File: .github/FEATURE_REQUEST.md (hunk 1 above)
Comment (xlanor) [reply .github/FEATURE_REQUEST.md]: Resolved!

**Comment by bwplotka** [comment]
Can you fix DCO? Check failure link below

**Comment by xlanor** [comment]
Forgot about that. dco signed off.

**Pull request merged**